src/property.ts

```diff
@@ -885,12 +909,14 @@ function Property$subArrayEvents(obj: Entity, property: Property, array: Observa
 	});
 }
 
-function Property$getInitialValue(property: Property): any {
+function Property$getInitialValue(property: Property, obj: Entity): any {
 	// Constant
 	if (property.isConstant)
 		return typeof property.constant === "function" ? (property.constant = property.constant()) : property.constant;
 
-	var val = property.defaultValue;
+	var val = property.initializer
+		? property.initializer.call(obj)
+		: property.defaultValue;
```

Comment:
Highlights the overlap between `default` (as it currently works) and `init`. Idea would be to diverge these such that `default` is used exclusively for _reactively_ updating a property.